pcli/src/command/validator.rs

```diff
@@ -31,6 +52,43 @@ impl ValidatorCmd {
 
                 println!("{}", ik);
             }
+            ValidatorCmd::UploadDefinition { file, fee, source } => {
+                // The definitions are stored in a JSON document,
+                // however for ease of use it's best for us to generate
+                // the signature here based on the configured wallet.
+                //
+                // TODO: eventually we'll probably want to support defining the
+                // identity key in the JSON file.
+                //
+                // We could also support defining multiple validators in a single
+                // file.
```

Comment:
maybe we should have a command with some command-line options that would generate the definition?